<?xml version="1.0"?>

<launch>
  <!-- <arg name="uri" default="radio://0/35/2M/E7E7E7E703" /> -->
  <arg name="uri" default="radio://0/80/2M/E7E7E7E704" />
  <!-- <arg name="uri" default="radio://0/57/2M/E7E7E7E705" /> -->

  <include file="$(find crazyflie_driver)/launch/crazyflie_server.launch">
  </include>

  <group ns="crazyflie4">
    <node pkg="crazyflie_driver" type="crazyflie_add" name="crazyflie_add" output="screen">
      <param name="uri" value="$(arg uri)" />
      <param name="tf_prefix" value="crazyflie4" />
      <param name="enable_logging" value="False" />
    </node>
  </group>

  <include file="$(find vicon_bridge)/launch/vicon.launch"/>

  <node name="test" pkg="crazyflie_demo" type="a_cooperative_quad.py" output="screen"> </node>

</launch>